doc/examples/segment_clustering_metrics.py

```diff
@@ -47,7 +47,11 @@ def get_streamlines():
 `Feature` is the `IdentityFeature` the streamlines won't be resampled thus
 saving some computational time.
 
-**Note:** Inputs must be sequences of same length.
+**Note:** Inputs must be sequences of same length. Also when the input
+feature of this metric ``is_order_invariant=True`` then the metric is
```

Comment:
That's misleading. It's not because the `Features` object has `is_order_invariant=True` that this metric becomes the MDF. However, it is true it would produce the same result.

That said, I don't see how you can design a `Feature` object that would allow you to produce the same results as the MDF described in `[Garyfallidis12]_`.